File: Otras_Evidencias/InputOutput.java
// Santiago de Jesús Bolio Cano
//540809

import java.util.Scanner; // Librería para ingreso de datos

class InputOutput{
    public static void main(String args[]){
        
        // NombreClase nombreObjeto = new NombreClase();
        Scanner     input = new   Scanner(System.in);
        
        String nombre;
        int n1, n2;
        
        System.out.println("¿cual es tu nombre?");
        nombre = input.nextLine(); // input() - cin>>
        
        System.out.println(" hola " + nombre+ " escribe dos valores enteros: ");
        n1 = input.nextInt(); //int(input())
        n2 = input.nextInt();
        
        System.out.println("la suma es: "+ (n1+n2));
    }
}

File: Otras_Evidencias/input output.cpp
// Santiago de Jesús Bolio Cano
//540809

#include <iostream>
using namespace std;
int main()
{
string nombre;
int n1, n2;
cout << ("¿cual es tu nombre?")<< endl;
cin >> nombre;
cout <<(" hola ", nombre, " escribe dos valores enteros: ");
cin >> n1;
cin >> n2;
cout<< "la suma es: " << (n1+n2);
    return 0;
}